lib/open_epic.rb

```diff
@@ -24,6 +24,10 @@ def initialize(fhir_base_url, client_id, options={})
   end
 
 
+  def test_method()
```

Comment:
Style/DefWithParentheses: Omit the parentheses in defs when the method doesn't accept any arguments.